pkgs/by-name/ap/apple-sdk/package.nix

```diff
@@ -0,0 +1,116 @@
+{
+  lib,
+  stdenv,
+  stdenvNoCC,
+  substitute,
+
+  # Specifies the major version used for the SDK. Uses `hostPlatform.darwinSdkVersion` by default.
+  darwinSdkMajorVersion ? (
+    if lib.versionOlder stdenv.hostPlatform.darwinSdkVersion "11" then
+      lib.versions.majorMinor stdenv.hostPlatform.darwinSdkVersion
+    else
+      lib.versions.major stdenv.hostPlatform.darwinSdkVersion
+  ),
+  # Enabling bootstrap disables propagation. Defaults to `false` (meaning to propagate certain packages and `xcrun`)
+  # except in stage0 of the Darwin stdenv bootstrap.
+  enableBootstrap ? stdenv.name == "bootstrap-stage0-stdenv-darwin",
```

Comment:
That other commit is so random packages in nixpkgs can use `darwin.bootstrapStdenv` in place of `stdenv`. This check is needed because each SDK is defined separately (as per best practices for different versions sharing the same derivation), so I’d have to manually override every single SDK in stage 0, which is ugly.

I should note I tried checking `__bootPackages`, but that caused evaluation errors when I ran it through the ofborg `outPaths.nix` check.